src/dayview.html

```diff
@@ -1,187 +1,181 @@
-          <init-position-scroll *ngIf="0!==currentViewIndex" class="dayview-normal-event-container"
-              [initPosition]="initScrollPosition">
-              <table class="table table-bordered table-fixed dayview-normal-event-table">
-                  <tbody>
-                      <tr *ngFor="let tm of views[0].rows; let i = index">
-                          <td class="calendar-hour-column text-center">
-                              {{hourColumnLabels[i]}}
-                          </td>
-                          <td class="calendar-cell">
-                              <ng-template [ngTemplateOutlet]="dayviewInactiveNormalEventSectionTemplate"
-                                  [ngTemplateOutletContext]="{tm:tm, hourParts: hourParts}">
-                              </ng-template>
-                          </td>
-                      </tr>
-                  </tbody>
-              </table>
-          </init-position-scroll>
-      </div>
-      <div class="swiper-slide slide-container">
-          <div class="dayview-allday-table">
-              <div class="dayview-allday-label">{{allDayLabel}}</div>
-              <div class="dayview-allday-content-wrapper scroll-content">
-                  <table class="table table-bordered dayview-allday-content-table">
-                      <tbody>
-                          <tr>
-                              <td class="calendar-cell"
-                                  [ngClass]="{'calendar-event-wrap':views[1].allDayEvents.length>0}"
-                                  [ngStyle]="{height: 25*views[1].allDayEvents.length+'px'}"
-                                  *ngIf="1===currentViewIndex">
-                                  <ng-template [ngTemplateOutlet]="dayviewAllDayEventSectionTemplate"
-                                      [ngTemplateOutletContext]="{allDayEvents:views[1].allDayEvents,eventTemplate:dayviewAllDayEventTemplate}">
-                                  </ng-template>
-                              </td>
-                              <td class="calendar-cell" *ngIf="1!==currentViewIndex">
-                                  <ng-template [ngTemplateOutlet]="dayviewInactiveAllDayEventSectionTemplate"
-                                      [ngTemplateOutletContext]="{allDayEvents:views[1].allDayEvents}">
-                                  </ng-template>
-                              </td>
-                          </tr>
-                      </tbody>
-                  </table>
-              </div>
+    <div *ngFor="let sliderIndex of sliderIndexList" class="swiper-slide slide-container dayview-slide-container">
+      <ng-container *ngIf="!dayviewShowCategoryView">
+        <div class="dayview-allday-table">
+          <div class="dayview-allday-label">{{allDayLabel}}</div>
+          <div class="dayview-allday-content-wrapper scroll-content">
+            <table class="dayview-table dayview-table-bordered dayview-allday-content-table">
```

Comment:
still add table in the class, as user on previous version may already customize based on ".table"

Reply:
you could use '.dayview-slide-container table' to add customized style